data/input_files/tests/nonlinear_solid/dyn_newmark_solve.lua

```diff
@@ -54,15 +54,15 @@ nonlinear_solid = {
 
     -- initial conditions
     initial_displacement = {
-        vec_coef = function (x, y, z)
-            return 0, 0, 0
-        end  
+        vec_coef = function (v)
+            return Vector.new(0, 0, 0)
```

Comment:
Will this work if it's just `Vector.new(0,0)`? The newmark problems are actually 2D.

Reply:
Yes - for reference, I've got the full set of permissible operations documented here: https://axom.readthedocs.io/en/develop/axom/inlet/docs/sphinx/advanced_types.html#adding-functions-to-a-schema